cross/fish/Makefile

```diff
@@ -0,0 +1,26 @@
+PKG_NAME = fish
+PKG_FULL_NAME = fish-shell
+PKG_EXT = tar.gz
+PKG_DOWNLOAD_METHOD = git
```

Comment:
Yep, I'd suggest to wait for the next release, which, hopefully, isn't too far away. For the record, `SPK_VERS` mentions `2.1.2`, and I assumed the git hash is a checkout for that release (right?)